tests/test-validate-app-receipt.js

```diff
@@ -75,6 +75,63 @@ describe('fxpay.validateAppReceipt()', function() {
     validator.finish();
   });
 
+  it('fails when receipt was issued by a disallowed store', function(done) {
+    // Disallow all other stores:
+    helper.appSelf.manifest.installs_allowed_from = [
```

Comment:
Does this get set in a `beforeEach`? It looks like this change could leak into the next test.